Projects/UOContent/Gumps/AdminGump.cs

```diff
@@ -2091,23 +2092,30 @@ public override void OnResponse(NetState sender, RelayInfo info)
                                     }
                                     else
                                     {
-                                        if (Core.ML)
+                                        if (Core.ML && Core.AvailableMaps.Includes(MapSelectionFlags.Malas))
```

Comment:
I am thinking it might make more sense to move it from `Core` to `Map`?

Reply:
seems reasonable!